server/views/pages/submitted.html

```diff
@@ -24,7 +24,7 @@
     {{
       govukButton({
         text: 'Continue to statement',
-        href: '/incidents/' + incidentId + '/statement',
+        href: '/incidents/' + reportId + '/statement',
```

Comment:
I'm guessing you're not changing the url just yet?

Reply:
no, in a future PR